src/Tests/Tests.csproj

```diff
@@ -16,8 +16,9 @@
     <PackageReference Include="ApprovalUtilities" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
-    <PackageReference Include="NServiceBus" Version="7.0.0" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
+    <PackageReference Include="NServiceBus" Version="7.1.0" />
```

Comment:
Why 7.1.0 here, when other PRs are going to 7.1.1?

Reply:
just wasn't on myget yet at that point. Since we had plenty of other packages going with 7.1 as well I didn't feel a strong need to wait for MyGet to offer the new package.